packages/shared/realm/sagas.ts

```diff
@@ -146,13 +144,7 @@ function* handleNewBFF() {
       // bind messages to this comms instance
       unbind = yield call(bindHandlersToBFF, action.payload, identity?.address)
 
-      // enable/disable minimap
-      const miniMapVisible =
-        !action.payload.about.configurations?.minimap || !action.payload.about.configurations?.minimap.enabled
       yield waitForRendererInstance()
```

Comment:
Also delete this one
```suggestion
      yield waitForRendererInstance()
```